packages/backend/src/clients/Google/GoogleDriveClient.ts

```diff
@@ -52,4 +49,44 @@ export class GoogleDriveClient {
             console.debug(`${file.name} (${file.id})`);
         });
     }
+
+    async appendToSheet(
+        refreshToken: string,
+        fileId: string,
+        csvContent: string,
+    ) {
+        if (!this.isEnabled) {
+            throw new Error('Google Drive is not enabled');
+        }
+        const authClient = await GoogleDriveClient.getCredentials(refreshToken);
+
+        const sheets = google.sheets({ version: 'v4', auth: authClient });
+        const sheetRows = csvContent.split('\n').map((row) => row.split(','));
+
+        // Clear first sheet before writting
```

Comment:
we could ask the user to specify a specific sheet (this would prevent the problem where a user reorders the sheet in the UI and then another sheet gets overwritten)

Reply:
let's create a ticket

Reply:
Created a new ticket in milestone